lua/catppuccin/groups/integrations/copilot_vim.lua

```diff
@@ -0,0 +1,10 @@
+local M = {}
+
+function M.get()
+	return {
+		CopilotSuggestion = { fg = C.subtext0 },
+		CopilotAnnotation = { fg = C.blue },
+	}
+end
```

Comment:
Could you please provide before/after screenshots of this change?

Reply:
Here is the after

<img width="492" alt="SCR-20241113-oflv" src="https://github.com/user-attachments/assets/ed5daa49-3081-49ec-a781-6920321a95bb">

I'm having a hard time getting the highlight group to reset, but I believe all text is set to a `#555555` color.